### PR TITLE
refactor: sponsorship forms page and bug fixes

### PR DESCRIPTION
**Pull request by csell5**
* adds apply now button on job posting
* adds sponsorship forms page /support/sponsors/forms/
* fixes staybridge typo

### CODE REVIEW

File: src/routes/partners/_components/_JobDetails.svelte [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <script>
 	export let jobListing;
 
+	console.log({ jobListing });
```
Comment (brettski) [inline src/routes/partners/_components/_JobDetails.svelte:4]: Did you want keep this in the production release?

Comment (brettski) [reply src/routes/partners/_components/_JobDetails.svelte:4]: @csell5 ^^

**Review by brettski — DISMISSED** [review]

File: src/routes/partners/_components/_JobDetails.svelte (hunk 1 above)
Comment (csell5) [reply src/routes/partners/_components/_JobDetails.svelte]: nope.. just pushed an update

**Review by brettski — APPROVED** [review]

**Pull request merged**